### DISCUSSION TITLE
The github student pack has no school email issues

### DISCUSSION THREAD

**Discussion by FangSuYu** [community/community#135038] [GitHub Education]
### Hi everyone,

# The github student pack has no school email issues
I am a current student, but the school does not have an education email (edu email), but there is a certificate of learning network, how can I pass the student certification
![微信图片_20240808140444](https://github.com/user-attachments/assets/6778cdd3-7884-4bca-9cd9-6b6f583df3e9)

**Comment by Xitee1** [comment]
Having the same issue, I also don't have a school email address (only teachers have) and am unable to get approved.

I tried making a manual request by not choosing my school in the search results and entering everything manually. I've got for lots of reasons, but the first one is the same as yours.

My other deny reasons were also kinda dumb. I had to downscale the image because it was too large (3mb, only 1mb is allowed). Then a reason why I was rejected was that the image has been edited :confounded: 

I also got rejected because my card is not from the school itself. Here in Germany when making a education, there is the `IHK` which stands in between the school and the training company and is there to manage the education process (hope I explained that correct in english). From that `IHK` I have a physical card that I can normally use for such verification things. But at GitHub it got rejected because it is not from the school.

Such a shame that Jetbrains switched to Github for the verification. Currently I still have access to all tools but the licenses run out soon. It was absolutely not problem before while Jetbrains still did the verification. Just send the image of the card, got access the next day.
